core/src/main/java/com/github/mizool/core/Identifiable.java

```diff
@@ -18,5 +18,13 @@
 
 public interface Identifiable<T extends Identifiable>
 {
-    Identifier<T> getIdentifier();
+    default Identifier<T> getIdentifier()
```

Comment:
Add an `@Deprecated` annotation plus an `@deprecated` Javadoc tag that says to use getId() instead.